docs/concepts/flink-architecture.md

```diff
@@ -169,7 +169,7 @@ A _Flink Application_ is any user program that spawns one or multiple Flink
 jobs from its ``main()`` method. The execution of these jobs can happen in a
 local JVM (``LocalEnvironment``) or on a remote setup of clusters with multiple
 machines (``RemoteEnvironment``). For each program, the
-[``ExecutionEnvironment``]({{ site.baseurl }}/api/java/) provides methods to
+[``ExecutionEnvironment``]({{ site.javadocs_baseurl }}/api/java/) provides methods to
```

Comment:
Why do we need to change this?
Seems the link is correct and reachable. (test use [documentation of 1.11](https://ci.apache.org/projects/flink/flink-docs-release-1.11/concepts/flink-architecture.html#flink-application-execution))

Reply:
It's kind of weird. I run `build_docs.sh` shell, and when I clicked this link in my local browser, it can not reachable. This is a javadocs url not other document, So I change this, and it is ok in my local. I checked the master and `1.11` just now, yes, it is reachable.
I run the `check_links.sh` just now, it also showed that ` ERROR `/api/java/' not found.`
BTW, there are other links broken by running  `check_links.sh` shell.
1.`/api/java/org/apache/flink/types/RowKind.html' not found.`
2.`/dev/python/table-api-users-guide/udfs.html`
3. `/dev/python/user-guide/table/dependency_management.html`
For the first, I checked it, it is the same with this.
For the second, I did not found any documents use this url.
For the third, Maybe it is really broken. We should use `dev/python/table-apis-users-guide/dependency_management.html` not `dev/python/user-guide/table/dependency_management.html`.

Reply:
@wangxlong I'm not very sure why the java doc links broken locally. maybe you need to generate the java doc locally before reaching to the doc.

Thanks for checking the broken links. Could you please file an issue to fix the other broken links you mentioned?

Reply:
Yes, I close this pr, and open a issue to fix these.